### ISSUE TITLE
Add a button to close the popup

### ISSUE THREAD

**Issue by xcoulon** [openshiftio/openshift.io#3066]
When changing the state of a work item, I get a popup to confirm that the change was taken into account, but I have to wait for a few seconds for it to disappear automatically, which is annoying, especially because this popup is partially over the button to close the work item panel:

<img width="505" alt="screenshot 2018-04-12 16 13 34" src="https://user-images.githubusercontent.com/178502/38682980-aab90656-3e6c-11e8-8063-865565956b8c.png">

**Comment by joshuawilson** [comment]
The `ESC` key also gets you out of the side panel view.

**Comment by nimishamukherjee** [comment]
@sunilmalagi please confirm the behaviour for notifications when a work item is updated.
Can we go ahead with @catrobson 's suggestion?

**Comment by rgarg1** [comment]
@sunilmalagi Are we also agreeing on providing a close option to dismiss the notification, if the users need to?

**Comment by rgarg1** [comment]
Even though its marked as SEV-3, it is such an irritant that I'd rather see it bumped to SEV-2 + priority0.

WDYT @qodfathr ?

**Comment by qodfathr** [comment]
@rgarg1 I agree it's super annoying, but I feel the severity is currently correct. Even if we bumped to SEV2, I would not assign p0.

But, if all of the Planner SEV2 p0's get completed, then I'd vote to move this to SEV2 and put it at the top of the stack. :)

**Comment by Veethika** [comment]
@xcoulon @Preeticp @rgarg1 @nimishamukherjee We should go ahead with @catrobson 's suggestion of showing notification only for serious actions and not when there's a very clear visual feedback following it.
And every notification should come with a 'close' option to avoid any hinderance.
Let me know if we all agree on this so that @sunilmalagi could take it further.

**Comment by Veethika** [comment]
UX solution provided, un-assigning myself.

**Comment by Veethika** [comment]
@rgarg1 @xcoulon removing UX label from this issue as the solution has been provided and awaits implementation.

**Comment by nimishamukherjee** [comment]
We now have notifications only for erros.

**Comment by christianvogt** [comment]
As per UX guidance, our notifications should include a close button.
